installer/install.iss.in

```diff
@@ -22,6 +23,7 @@ LZMAUseSeparateProcess=yes
 OutputBaseFilename={#APP_NAME+'-'+APP_VERSION}
 OutputDir={#GetEnv('USERPROFILE')}
 SolidCompression=yes
+SourceDir={#GetEnv('ROOTDIR')}

```

Comment:
Theoretically, we could use absolute paths, but that would be just a lot of churn for no gain. So I agree this is the most elegant solution!